src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -1,10 +1,9 @@
-import _ from 'lodash/fp'; // eslint-disable-line no-restricted-imports
 import Scroll from 'react-scroll';
 
-export function focusElement(selectorOrElement, options) {
+export function focusElement(selectorOrElement, options, testDom) {
   const el =
     typeof selectorOrElement === 'string'
-      ? document.querySelector(selectorOrElement)
+      ? (testDom || document).querySelector(selectorOrElement)
```

Comment:
Instead of adding the extra argument, could the test get away with setting `global.document`?

Reply:
Yeah, that would work as well... should we modify the `global.document` though? I've noticed `window.Forms` and `windows.VetsGov` are defined as global variables. Should I use that?

Reply:
Ok, I switch to use `window.Forms.document` - the variable name probably isn't ideal since we're passing a fragment, but it works.